File: docs-old-1103-2/data-tools/codesnippet/VisualBasic/query-datasets_5.vb
        Dim orderID As Integer = 10707
        Dim customer As NorthwindDataSet.CustomersRow

        customer = CType(NorthwindDataSet.Orders.FindByOrderID(orderID).
            GetParentRow("FK_Orders_Customers"), NorthwindDataSet.CustomersRow)

        MessageBox.Show(customer.CompanyName)

File: docs/data-tools/codesnippet/CSharp/query-datasets_5.cs
            int orderID = 10707;
            NorthwindDataSet.CustomersRow customer;

            customer = (NorthwindDataSet.CustomersRow)northwindDataSet.Orders.
                FindByOrderID(orderID).GetParentRow("FK_Orders_Customers");

            MessageBox.Show(customer.CompanyName); 